examples/specs/window_transform_row_number.vl.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Why is this example important?  
(Is it important for explaining any concept in the documentation?)

Reply:
I'd say rank chart as in https://stats.stackexchange.com/a/178344  is a more compelling use case of `row_number`

Reply:
I dropped this file, and added a new one which compares the ranks of the top 5 students on an exam.